modules/nf-core/gatk4/createsequencedictionary/main.nf

```diff
@@ -20,7 +20,7 @@ process GATK4_CREATESEQUENCEDICTIONARY {
     script:
     def args = task.ext.args ?: ''
 
-    def avail_mem = 6
+    def avail_mem = 6144
```

Comment:
Why so much for such a small process

Reply:
`avail_mem` was accidentally decreased from 6GB to 6MB which - as you may already know - caused some problems over in Sarek. I guess the purpose of this PR is just to get things working again; not to find the lowest still-working value of `avail_mem`.

Reply:
yes well aware of the problem, just questioning why has 6GB by default assigned in comparison to all others. two birds one stone and so on :)

Reply:
The picard equivalent only has 3GB. Also, and I've said this before, why do we have duplicate modules?